less/_grid.less

```diff
@@ -76,9 +76,13 @@
 
 // other CSS for the container of the new columns
 .grid {
+	@grid-sides: 15px;
 	.clearfix;
 
-	width: 100%;
+	padding: 0 25px 40px 25px;
```

Comment:
The grid has the padding in order to give some space between rows. I may need to rework this though so we don't have any when we only have one row.

The padding in the side might be an issue though. We have padding on the sides of our interfaces, but I might want to take another look at this.